Sources/Layouts/LabelLayout.swift

```diff
@@ -16,9 +16,11 @@ open class LabelLayout<Label: UILabel>: BaseLayout<Label>, ConfigurableLayout {
     open let text: Text
     open let font: UIFont
     open let numberOfLines: Int
+    open let lineHeight: CGFloat
 
     public init(text: Text,
                 font: UIFont = LabelLayoutDefaults.defaultFont,
+                lineHeight: CGFloat? = nil,
```

Comment:
please add this to the convenience initializers too

Reply:
Added `lineHeight` parameter to convenience initializers.